Source/FakeItEasy.Specs/TaskSpecs.cs

```diff
@@ -0,0 +1,184 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Xbehave;
+
+    public static class TaskSpecs
+    {
+        public interface IFooBarService
+        {
+            Task CommandAsync();
+
+            Task<int> QueryAsync();
+
+            Task<Foo> GetFooAsync();
+
+            Task<Bar> GetBarAsync();
+        }
+
+        [Scenario]
+        public static void ConfiguredAsyncMethodWithReturnValueOfTypeInt(

```

Comment:
I'm leaning toward removing this scenario and the other `Configured*` ones, because they just test that the method returns the configured value, which is already tested in many other places. What do you think?